2021-09-22 16:32:42,419 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
2021-09-22 16:32:42,419 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот запущен!
2021-09-22 16:32:43,763 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:32:49,353 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:32:54,895 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:33:00,412 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:33:16,090 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
2021-09-22 16:33:16,091 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот запущен!
2021-09-22 16:33:16,981 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:33:22,484 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:33:28,061 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:33:33,714 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:33:39,303 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:33:45,405 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:34:24,097 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
2021-09-22 16:34:24,097 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот запущен!
2021-09-22 16:34:27,969 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:34:28,112 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот упал (ну вообще - не упал, а просто ошибка) с ошибкой: Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:34:33,855 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:34:33,995 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот упал (ну вообще - не упал, а просто ошибка) с ошибкой: Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:34:39,789 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:34:39,930 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот упал (ну вообще - не упал, а просто ошибка) с ошибкой: Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:37:53,766 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
2021-09-22 16:37:53,767 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот запущен!
2021-09-22 16:37:54,872 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:39:24,135 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
2021-09-22 16:39:24,135 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот запущен!
2021-09-22 16:39:25,079 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:40:43,450 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
2021-09-22 16:40:43,450 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот запущен!
2021-09-22 16:40:44,336 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:40:50,091 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:41:07,521 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
2021-09-22 16:41:07,521 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот запущен!
2021-09-22 16:41:08,493 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:41:14,088 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:43:31,251 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
2021-09-22 16:43:31,251 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот запущен!
2021-09-22 16:43:32,357 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:43:37,903 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:43:43,474 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:44:41,438 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
2021-09-22 16:44:41,438 - [INFO] - __main__ - (homework.py).send_message(104) - Бот отпрвил сообщение: Бот запущен!
2021-09-22 16:44:42,359 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:44:47,911 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:44:53,431 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:44:58,987 - [CRITICAL] - __main__ - (homework.py).get_homeworks(88) - Сервер отвалился (код ответа страницы) - 404, при переходе поhttps://praktikum.yandex.ru/api/user_api/homework_statuses/
2021-09-22 16:53:12,956 - [DEBUG] - __main__ - (homework.py).main(156) - Бот запущен!
